StyleCop.Analyzers/StyleCop.Analyzers/Helpers/TypeDeclarationSyntaxHelper.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace StyleCop.Analyzers.Helpers
+{
+    using System.Linq;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.CSharp;
+
+    internal class TypeDeclarationSyntaxHelper
```

Comment:
:bulb: `static class`

:question: SA1400 uses this for updating modifiers of both types and members. Should it have a different name?